oada-core/http-handler/oada-srvc-http-handler/tsconfig.json

```
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "ES2018",
+    "strict": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "module": "CommonJS",
+    "lib": ["ES2018"],
+    "sourceMap": true,
+    "outDir": "dist",
+    "baseUrl": ".",
+    "paths": {
+      "*": ["node_modules/*"]
+    }
+  },
+  "include": ["*.ts"],
+  "exclude": ["node_modules"]
+}
```

```suggestion
  {
    "compilerOptions": {
      "module": "commonjs",
      "moduleResolution": "node",
      "target": "es2019",
      "lib": ["ES2019"],
      "strict": true,
      "noUnusedLocals": true,
      "noUnusedParameters": true,
      "noFallthroughCasesInSwitch": true
    }
  }
```

For some reasons `noUnusedLocals`, `noUnusedParameters`, `noFallthroughCasesInSwitch` are not on by default with strict mode. The later being more of a linting issue, but I see the first two as a code bugs.

---

I can maybe agree on the first one, but extra parameters are also just lint.

---

I added the first two.